common/thorhelper/thorsoapcall.cpp

```diff
@@ -1371,6 +1402,86 @@ class CWSCAsyncFor : implements IWSCAsyncFor, public CInterface, public CAsyncFo
         }
     }
 
+    bool checkContentEncodingType(const char* encoding)
```

Comment:
Function is gzip specific.  checkContentEndcodingGzip or checkContentEndcodingTypeGzip might be better.